openapi/components/schemas/ReadyToPay/Methods/ReadyToPayPaymentCardMethod.yaml

```
@@ -12,8 +12,9 @@ properties:
     type: object
     nullable: true
     description: |-
-      Specific feature (for example, a digital wallet) of this method.
-      If method doesn't have any features, the value is null.
+      Specific feature of this method.
+      For example, a digital wallet or a processor.
```

This doesn't make sense. The only valid example is a digital wallet.
```suggestion
      For example, a digital wallet.
```